metrics/worker/main.go

```diff
@@ -55,6 +55,8 @@ func newPrometheusScope(c prometheus.Configuration) tally.Scope {
 		Separator:       prometheus.DefaultSeparator,
 		SanitizeOptions: &sanitizeOptions,
 		Prefix:          "temporal_samples",
+		Tags:            map[string]string{"my_key1": "my_value1",
+			                       "my_key2": "my_value2"},
```

Comment:
You need to run `go fmt` on this file. All Go IDEs should do this automatically for you on save.

Reply:
@cretz done